OpenOversight/app/templates/sort.html

```diff
@@ -13,36 +35,48 @@
       </div>
 
       <div class="row">
-        <div class="frontpage-leads">
-          <div>
-            <img class="center-block img-responsive" src="{{ path }}" alt="Picture to be sorted">
-          </div>
-        </div>
-      </div>
-
-      <div class="row">
         <div class="col-sm-4 text-center">
           <form action="{{ url_for('main.classify_submission', image_id=image.id, contains_cops=0) }}" method="post">
-            <button type="submit" name="action" class="btn btn-lg btn-danger">
+            <button type="submit" name="action" id="answer-no" class="btn btn-lg btn-danger">
               <span class="glyphicon glyphicon glyphicon-remove" aria-hidden="true"></span>
               No
             </button>
           </form>
         </div>
         <div class="col-sm-4 text-center">
-          <a href="{{ url_for('main.sort_images') }}" class="btn btn-lg btn-primary" role="button">
+          <a href="{{ url_for('main.sort_images') }}" id="answer-skip" class="btn btn-lg btn-primary" role="button">
             <span class="glyphicon glyphicon-repeat" aria-hidden="true"></span> Skip</a>
         </div>
         <div class="col-sm-4 text-center">
           <form action="{{ url_for('main.classify_submission', image_id=image.id, contains_cops=1) }}" method="post">
-            <button type="submit" name="action" class="btn btn-lg btn-success">
+            <button type="submit" name="action" id="answer-yes" class="btn btn-lg btn-success">
               <span class="glyphicon glyphicon glyphicon-ok" aria-hidden="true"></span>
               Yes!
             </button>
           </form>
         </div>
       </div>
 
+      <div class="row">
+        <div class="col-sm-4 text-center">
+          (shortcut: n)
+        </div>
+        <div class="col-sm-4 text-center">
+          (shortcut: s)
+        </div>
+        <div class="col-sm-4 text-center">
+          (shortcut: y)
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="frontpage-leads">
+          <div>
+            <img class="center-block img-responsive" src="{{ path }}" alt="Picture to be sorted">
+          </div>
+        </div>
+      </div>
+
```

Comment:
If you think that'd be clear enough to the user, I can make that change.

Options:

1. Tell the user about the shortcuts in a pop-up once (but only on desktop), and store in a cookie (or in their account profile, but not exposed in the UI) that they've seen it and know (so we don't show them again
    - Sounds complicated
2. Underline first letter of each + mention these shortcuts in the FAQ
3. Underline first letter of each + mention shortcuts elsewhere (blurb when they first log in?)